MultiSlider.js

```diff
@@ -385,6 +327,48 @@ export default class MultiSlider extends React.Component {
     if (positionOne !== prevPositionOne || positionTwo !== prevPositionTwo) {
       this.props.onMarkersPosition([positionOne, positionTwo]);
     }
+
+    if (this.state.onePressed || this.state.twoPressed) {
+      return;
+    }
+
+    let nextState = {};
+    if (
+      prevProps.min !== this.props.min ||
+      prevProps.max !== this.props.max ||
+      prevProps.step !== this.props.step ||
+      prevProps.values[0] !== this.state.valueOne ||
+      prevProps.sliderLength !== this.props.sliderLength ||
```

Comment:
Should this not also compare to this.props.values[0]?

Reply:
Good call sounds better indeed